src/AspNet.Security.OAuth.EVEOnline/project.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": "1.0.0-*",
```

Comment:
Added, just as in one of previous comment, yeoman generates project.json with version 1.0.0-* by default.